### PR TITLE
docs: DEV-2937: Update LSF guide

### PR DESCRIPTION
**Pull request by nicholasrq**

### CODE REVIEW

File: docs/source/guide/frontend.md [hunk 1]
```diff
@@ -6,150 +6,282 @@ meta_title: Customize User Interface
...
 
-LSF is located as a separate GitHub repository: 
-https://github.com/heartexlabs/label-studio-frontend
+LSF can be customized and extended to fit your needs, and you can use a custom version of LSF standalone or in your LS installation. For more information see [Custom LSF + LS integration](#custom-lsf-in-label-studio).
+
+LSF is located as a separate GitHub repository: https://github.com/heartexlabs/label-studio-frontend
 
 <br>
-<div style="margin:auto; text-align:center;"><img src="/images/LSF-modules.png" style="opacity: 0.9"/></div>
+<div style="margin:auto; text-align:center;"><img src="/images/frontend/lsf-in-ls.jpg" style="opacity: 0.9"/></div>
```
Comment (anithamuthu2022) [inline docs/source/guide/frontend.md:16]: why is the image a .jpg file instead of a .png file?

File: docs/source/guide/frontend.md [hunk 2]
```diff
@@ -6,150 +6,282 @@ meta_title: Customize User Interface
...
 
+## Installation
+There are two ways to install the LSF: via package manager or via Unpkg CDN.
```
Comment (anithamuthu2022) [inline docs/source/guide/frontend.md:19]: There are two ways to install the LSF as follows: 

1. Using the package manager 
2. Using the Unpkg CDN

File: docs/source/guide/frontend.md (hunk 1 above)
Comment (nicholasrq) [reply docs/source/guide/frontend.md:16]: png is good when dealing with transparent background. overall, compression and size-wise jpg is better with all internal optimizations

Comment (anithamuthu2022) [reply docs/source/guide/frontend.md:16]: https://heartex.atlassian.net/wiki/spaces/ENGIN/pages/824737793/Label+Studio+Documentation+Handbook#Add-images

**Pull request merged**